src/components/game/Board.jsx

```diff
@@ -1,26 +1,10 @@
-import React, { useState } from "react";
+import React from "react";
```

Comment:
If you don't need other modules from React, you can remove this import altogether. This is no longer needed with functional components after v17.